src/SimpleBayesianConsensusCaller.cpp

```diff
@@ -288,10 +292,10 @@ void SimpleBayesianConsensusCaller::loadConfiguration(ifstream& matrixFile){
             if (tokens[0] == "Name"){
                 parseName(matrixFile, line);
 
-            }else if (tokens[1] == "prior"){
+            }else if (tokens.size() > 1 and not tokens[0].empty() and tokens[1] == "prior"){
                 parsePrior(matrixFile, line, tokens);
 
-            }else if (tokens[1] == "likelihood"){
+            }else if (tokens.size() > 1 and not tokens[0].empty() and tokens[1] == "likelihood"){
                 parseLikelihood(matrixFile, line, tokens);
             }
```

Comment:
Should we also add a check for an extraneous, not recognized line?

Reply:
I thought about this, and I also considered adding an initial check to make sure that each expected header is found only once, using a set. It's up to you, I think. Maybe a conventional config parser library would have been best

Reply:
Yes, at some point perhaps a overhaul of this parsing code will be called for. Let's not worry about this for now. 

I will merge this PR in a few days, after the upcoming release.